security/zero_trust_workload_identity_manager/zero-trust-manager-upstream-authority.adoc

```diff
@@ -0,0 +1,40 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="zero-trust-manager-upstream-authority-plugins"]
+= Zero Trust Workload Identity Manager upstream authority plugins
+
+include::_attributes/common-attributes.adoc[]
+:context: zero-trust-manager-overview
+
+toc::[]
+
+:FeatureName: Zero Trust Workload Identity Manager
+include::snippets/technology-preview.adoc[]
+
+Upstream authority plugins are components that allow the SPIRE server to integrate with an existing Public Key Infrastructure (PKI) to obtain intermediate signing certificates. The SPIRE server then uses these intermediate certificates to cryptographically sign the {svid-full} that it issues to workloads. The plugins also allow the SPIRE server to use a pre-existing root of trust, rather than establishing a new, isolated one.
```

Comment:
```suggestion
Upstream authority plugins are components that allow the SPIRE server to integrate with an existing Public Key Infrastructure (PKI) to obtain intermediate signing certificates. The SPIRE server then uses these intermediate certificates to cryptographically sign the {svid-full} that it issues to workloads. The plugins also allow the SPIRE server to use a pre-existing root of trust, rather than establishing a new, isolated root of trust.
```